modules/tinymce/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 
 ### Added
+- Updated `applyFormat` and `removeRngStyle` to include the last item in the selection #TINY-6567
```

Comment:
- [ ] Since this is a fix for a customer reported bug it should be placed under `Fixed` and maybe be describing the bug fix.

So something like `Fixed a bug where it wasn't possible to align nested list items` maybe the editor team has a better suggestion here.

Generally the changelog should not include things that are internal. So if the change is not in the tinymce api directory then it's an internal change. The changelog is to be consumed by our users/customers so they need to be able to understand from their perspective what has been changed.